products/opensearch.md

```diff
@@ -14,8 +14,8 @@ releases:
 -   releaseCycle: "2"
     eol: false
     releaseDate: 2022-05-18
-    latest: "2.5.0"
-    latestReleaseDate: 2023-01-12
+    latest: "2.6.0"
```

Comment:
Maybe we should update the "Official Release schedule" on https://endoflife.date/opensearch ?